app/views/unattended/report_templates/host_-_installable_errata_count.erb

```diff
@@ -0,0 +1,38 @@
+<%#
+name: Host - Installable Errata Count
+snippet: false
+template_inputs:
+- name: Hosts filter
+  required: false
+  input_type: user
+  description: Limit the report only on hosts found by this search query. Keep empty
+    for report on all available hosts.
+  advanced: false
+  value_type: search
+  resource_type: Host
+  hidden_value: false
+model: ReportTemplate
+require:
+- plugin: katello
+￼ version: 4.9.0
+-%>
+<%- report_headers 'Host', 'Security Errata Count', 'Bugfix Errata Count', 'Enhancement Errata Count', 'Upgradable RPM Count' -%>
+<%- load_hosts(search: input('Hosts filter'), preload: [:content_facet]).each_record do |host| -%>
+<%- content_facet = host_content_facet(host) -%>
+<%-   
+   security_errata_count = content_facet.try(:installable_security_errata_count)
+   bugfix_errata_count = content_facet.try(:installable_bugfix_errata_count)
+   enhancement_errata_count = content_facet.try(:installable_enhancement_errata_count)
+   upgradable_rpm_count = content_facet.try(:upgradable_rpm_count)
```

Comment:
```suggestion
   security_errata_count = content_facet ? content_facet.installable_security_errata_count : 0
   bugfix_errata_count = content_facet ? content_facet.installable_bugfix_errata_count : 0
   enhancement_errata_count = content_facet ? content_facet.installable_enhancement_errata_count : 0
   upgradable_rpm_count = content_facet ? content_facet.upgradable_rpm_count : 0
```

This resolved it for me

Reply:
@wbclark First of all apologies for the delay, as I was caught up in multiple other tasks. Thanks for the suggestions, I amended the changes that you suggested. When you can please take a look again.